backend/Cargo.toml

```
@@ -14,12 +14,14 @@ axum = { version = "0.7", features = ["macros"] }
 hyper = { version = "1", features = ["full"] }
 tokio = { version = "1", features = ["full"] }
 tower = "0.4"
+tower-http = { version = "0.5.0", features = ["fs"] }
 utoipa = { version = "4.2", features = ["axum_extras"] }
 utoipa-swagger-ui = { version = "7.1.0", features = ["axum"], optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 sqlx = { version = "0.7", features = ["runtime-tokio", "sqlite", "chrono"] }
 chrono = { version = "0.4", features = ["alloc", "std", "serde"] }
+clap = { version = "4.0", features = ["derive"] }
```

Clap pulls in quite some sub dependencies. This could be a quick win to eliminate some. If the interface stays as simple as it is now, we could write our own CLI in the future and lose Clap altogether. 
```suggestion
clap = { version = "4.0", default-features = false, features = ["derive", "std"] }
```